From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Eneas U de Queiroz <cote2004-github@yahoo.com>
Date: Thu, 27 Sep 2018 08:29:21 -0300
Subject: Do not use host kernel version to disable AFALG

This patch prevents the Configure script from using the host kernel
version to disable building the AFALG engine on openwrt targets.

Signed-off-by: Eneas U de Queiroz <cote2004-github@yahoo.com>

--- a/Configure
+++ b/Configure
@@ -1810,7 +1810,9 @@ $config{CFLAGS} = [ map { $_ eq '--ossl-
 
 unless ($disabled{afalgeng}) {
     $config{afalgeng}="";
-    if (grep { $_ eq 'afalgeng' } @{$target{enable}}) {
+    if ($target =~ m/openwrt$/) {
+        push @{$config{engdirs}}, "afalg";
+    } elsif (grep { $_ eq 'afalgeng' } @{$target{enable}}) {
         push @{$config{engdirs}}, "afalg";
     } else {
         disable('not-linux', 'afalgeng');
